Addind shell script keys creator for DXL (#5655)
demisto/content: Integrations/McAfee_DXL/create_keys/create_and_upload_RSA_keys_for_DXL.md

## how to configure DXL based integration
- create RSA key pair.
    - make sure that **openssl** is installed
    - open a new directory.
    - download the sh script https://github.com/demisto/content/raw/master/Integrations/McAfee_DXL/create_keys.sh.zip and move to the directory.

>     - unzip the file run.

Reviewer: mention here what files should be created after you run the file.

    - fill out the required fields except the challenge password and the optional company name (leave it empty).

> - upload the public key.

Reviewer: add the file name of the public key

Author: shown down.

    ![go to serevr settings](https://github.com/demisto/content/raw/dxl_exemple/Integrations/McAfee_DXL/create_keys/img/go_to_serevr_settings.png)
    - under DXL certificates (Third Party) click edit.
    ![click edit](https://github.com/demisto/content/raw/dxl_exemple/Integrations/McAfee_DXL/create_keys/img/click_edit.png)
    - download the brokers certificate.
    ![export Broker certificates](https://github.com/demisto/content/raw/dxl_exemple/Integrations/McAfee_DXL/create_keys/img/export_Broker_certificates.png)
    - download the brokers list.
    ![export Brocker list](https://github.com/demisto/content/raw/dxl_exemple/Integrations/McAfee_DXL/create_keys/img/export_Brocker_list.png)
    - click import and select `client.crt` file click ok and upload it.
    ![click import](https://github.com/demisto/content/raw/dxl_exemple/Integrations/McAfee_DXL/create_keys/img/click_import.png)
    ![select client.crt file](https://github.com/demisto/content/raw/dxl_exemple/Integrations/McAfee_DXL/create_keys/img/select_client.crt_file.png)
    ![click ok](https://github.com/demisto/content/raw/dxl_exemple/Integrations/McAfee_DXL/create_keys/img/click_ok.png)
    - click save.

>     ![click save](https://github.com/demisto/content/raw/dxl_exemple/Integrations/McAfee_DXL/create_keys/img/click_save.png)

Reviewer: for every certificate related field(of the 3 fields) with the name of the field and what file contents should fill it.

Author: will be in the configure integration description.

> - test the integration (it may take a while before the key will be enable).

Reviewer: ```suggestion
- test the integration (it may take a few minutes until the key is enabled).```
